src/manage-apps/manage-repos-repoman.md

```diff
@@ -0,0 +1,92 @@
+# Managing Repositories with Repoman
+
+Add, remove, and edit repositories using Repoman. Repoman allows users to add Debian repositories, including Personal Package Archives (PPAs), and Flatpak repositories. Users may want to add a Debian repository or additional Flatpak repository if their software is not included in the Pop!\_Shop, or they may want to remove PPA’s after uninstalling a package.
+
+Access Repoman by launching the Pop!\_Shop, clicking the three lines, and then clicking `System Software Sources...`.
+
+![Access Repoman](/images/manage-repos/access-repoman.png)
+
+>**Caution**: Adding 3rd-party Debian or Flatpak repositories allows installation of software that has not been validated by System76 or other trusted Linux communities. Software available via 3rd-party Debian and Flatpak repositories may not be vetted against packages that introduce security vulnerabilities. Users should take caution and only add external repositories that they trust.
+---
+
+## Official Sources
+
+This section lists System76's official Pop!\_OS mirror. This mirror includes installation files and updates for all applications included in a default Pop!\_OS installation. This is also where installation files for many applications in the Pop!\_Shop reside. Changing this mirror is not necessary since Pop!\_OS will automatically download updates from the fastest source using System76's worldwide CDN.
+
+The toggle switches enable or disable software components based on licensing and support:
+
+- **Restricted** - Proprietary drivers and components that are not open source
+- **Universe** - Free and open-source software that is community maintained
+- **Multiverse** - Software restricted by copyright or legal reasons
+
+![Official Sources Tab](/images/manage-repos/official-sources-tab.png)
+
+## Managing Debian Repositories with Repoman
+
+Debian archives (including PPAs), are created by developers to distribute software not included in default Debian repositories.
```

Comment:
> Debian archives (including PPAs),

You can't have parenthesis and a comma here. Either remove the parenthesis and add a comma after "archives" too, or remove the comma.

I would say "in the default Pop!_OS and Ubuntu repositories" to avoid confusion; you're talking about Debian-format repositories, but this could also sound like Debian's own repositories (which we don't use; Ubuntu replaces them.) You could technically also just say "in the default Pop!_OS repositories" since that includes everything we configure by default, although it obfuscates the fact that we're based on Ubuntu (and Ubuntu curates/maintains most of the packages that are not installed by default.)

Reply:
Changed wording and fixed comma, also changed all mention of "Debian" to "apt" repository.